app/containers/PromptContainer.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react'
+import Prompt from '../components/Prompt'
+
+class PromptContainer extends Component {
+  constructor (props, context) {
+    super (props, context)
+    
+    this.state = {
+      username: '',
+      errorMessage: null
+    }
+    context.router
+
+    this.handleSubmitUser = this.handleSubmitUser.bind(this)
+    this.handleUpdateUser = this.handleUpdateUser.bind(this)
+  }
+
+  handleSubmitUser (e) {
+    e.preventDefault()
+    const username = this.state.username
```

Comment:
I would put move stuff around and add some space here for readability:
```
handleSubmitUser (e) {
    const {username} = this.state

    e.preventDefault()

    if (username === '') {
```

I'd also put the positive condition first. Technically you've done that, but since `username === ''` is the same as `!username` it's actually checking the false condition first.